src/app.ts

```diff
@@ -0,0 +1,41 @@
+import express, { Response, Application } from 'express';
+import mongoose from 'mongoose';
+
+import passport from 'passport';
+
+import config from './config';
```

Comment:
Do we need any error handling if the config file is not present?

Reply:
Umm, I have an example config file and I will add in the instructions to create config in the rEADME, I don't see a reason to add error handling for this.

Reply:
Okay. 
But I think that a 'file not found' error doesn't provide the developer with enough information. That's why if we had some config store, and if that import failed we could throw an appropriate detailed error message. And that would have to be handled only in one place(where the store is initialised).

Reply:
Got it, I will update it so that it throws an appropriate error for this.

Reply:
This is the error it throws for now. Is this fine?
```
 6 import config from './config';
                     ~~~~~~~~~~

    at createTSError (/home/sparsh/Desktop/Sparsh/Projects/alcoding/posterior/node_modules/ts-node/src/index.ts:434:12)
    at reportTSError (/home/sparsh/Desktop/Sparsh/Projects/alcoding/posterior/node_modules/ts-node/src/index.ts:438:19)
    at getOutput (/home/sparsh/Desktop/Sparsh/Projects/alcoding/posterior/node_modules/ts-node/src/index.ts:578:36)
    at Object.compile (/home/sparsh/Desktop/Sparsh/Projects/alcoding/posterior/node_modules/ts-node/src/index.ts:775:32)
    at Module.m._compile (/home/sparsh/Desktop/Sparsh/Projects/alcoding/posterior/node_modules/ts-node/src/index.ts:858:43)
    at Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Object.require.extensions.(anonymous function) [as .ts] (/home/sparsh/Desktop/Sparsh/Projects/alcoding/posterior/node_modules/ts-node/src/index.ts:861:12)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
[nodemon] app crashed - waiting for file changes before starting...

```

Reply:
No, this error message is not informative enough. Let's open an issue to solve this config feature.